blogposts/2020/can-you-hear-me-no-but-we-read-you.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Can you hear me? No, but we read you—an experiment with video calls"
+author: Quinn Keast
+authorUrl: https://twitter.com/quinnkeast
+publishDate: 2020-10-15
+tags: [blog]
+slug: can-you-hear-me-no-but-we-read-you-an-experiment-with-video-calls
+heroImage: /blog/silent-team-call.png
+published: true
+---
+
+At Sourcegraph, we’re all-remote. That means we write a lot—in emails, Google Docs, Github, and Slack. We work hard to cultivate a written-first culture.
+
+And that works beautifully for me—because, you see, I have a hearing problem. Or as I like to say, I’m deaf as a post.
+
+While I can hear sound, [I can’t turn that sound into words](https://quinnkeast.com/readme#errata), and rely on lip reading for day-to-day conversations.
+
+For me, a product designer, this approach to working is amazing. Long stretches of focus time and being able to decide when and how to best do my work. And for someone with hearing loss, this almost inherently creates a more inclusive environment.
+
+Still, from time to time, we rely on synchronous methods of communication: team syncs to catch up on a more personal level and bring teams together, moderated user research, and facilitated collaborative activities and workshops. More often than not, this means a video call.
+
+Video calls have always been a challenge for me. Lip reading doesn’t work well over video, because lip reading relies on a whole lot more visual information than just the lips, and video calls don’t carry the full visual and _emotional_ bandwidth needed to read lips easily. So instead, I use a series of hacks or built-in tools to help me out by providing real-time speech-to-text.
+
+These hacks and tools work well until things go wrong. Sometimes they’ll cut out. Sometimes they’ll lag and fall behind. When things go wrong, it can be challenging. Simple things like someone asking for my thoughts on something can create awkward moments of silence before I realize that someone said my name, because I was relying on a transcript that was five seconds behind.
```

Comment:
```suggestion
These hacks and tools work well, but sometimes they’ll cut out or they’ll lag and fall behind. And then simple things like someone asking for my thoughts on something can create awkward moments of silence before I realize that someone said my name, because I was relying on a transcript that was five seconds behind.
```

(Since you do the "Things work well until they don't" further down, I don't think it should be used twice)